app/javascript/components/applications/ApplicationEditPage.js

```diff
@@ -8,6 +8,7 @@ import { saveApplication } from './actions'
 
 const ApplicationEditPageForm = ({ listing, application, editPage, lendingInstitutions }) => {
   const saveEditApplication = async (submitType, submittedValues, listing, editPage) => {
+    submittedValues.listing_id = listing.id
```

Comment:
This is sort of a major problem still left with these updates: what to do with the listing Id. We have to send it, and in some cases it's stored in the application object as:
"listing": {
"id": "blah"
}

and in some cases it's stored as listing_id. I've been trying to standardize around listing_id because that is the only piece that comes with the application and is saved with the application. For other listing information, I'd rather keep it out of the listing object.